externs/pictureinpicture.js

```diff
@@ -34,6 +34,10 @@ HTMLDocument.prototype.pictureInPictureElement;
 HTMLDocument.prototype.pictureInPictureEnabled;
 
 
+/** @type {Element} */
+HTMLDocument.prototype.polyfillPictureInPictureElement;
```

Comment:
IMO, this doesn't belong in the externs.  The externs should represent what's in the browser environment already.